pom.xml

```diff
@@ -17,81 +17,98 @@
 		<test.file></test.file>
 		<config.file>default</config.file>
 		<applicationUrl>http://www.google.com</applicationUrl>
-		<surefire.fork.count>5</surefire.fork.count>
-		<cucumber.version>1.1.2</cucumber.version>
-		<selenium.version>2.31.0</selenium.version>
-		<junit.version>5.0</junit.version>
+		<selenium.version>4.1.0</selenium.version>
+		<testng.version>7.4.0</testng.version>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
+		<config.file>config/sample.testng.xml</config.file>
 	</properties>
 
 	<dependencies>
 		<dependency>
 			<groupId>org.testng</groupId>
 			<artifactId>testng</artifactId>
-			<version>6.10</version>
+			<version>${testng.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
-			<version>2.5</version>
+			<version>2.11.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
-			<version>2.53.0</version>
-		</dependency>
-		<dependency>
-			<groupId>com.browserstack</groupId>
-			<artifactId>browserstack-local-java</artifactId>
-			<version>1.0.3</version>
-		</dependency>
-		<dependency>
-			<groupId>com.googlecode.json-simple</groupId>
-			<artifactId>json-simple</artifactId>
-			<version>1.1.1</version>
+			<version>${selenium.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>com.codeborne</groupId>
 			<artifactId>selenide</artifactId>
-			<version>4.2</version>
+			<version>6.7.2</version>
 			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
-			<version>1.5.6</version>
+			<version>2.0.0</version>
 			<type>jar</type>
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-simple</artifactId>
-			<version>1.5.6</version>
+			<version>2.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.browserstack</groupId>
+			<artifactId>browserstack-java-sdk</artifactId>
+			<version>LATEST</version>
+			<scope>compile</scope>
 		</dependency>
 	</dependencies>
 
 	<build>
 		<plugins>
+			<plugin>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>getClasspathFilenames</id>
+						<goals>
+							<goal>properties</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.19.1</version>
+				<configuration>
+					<suiteXmlFiles>
+						<suiteXmlFile>${config.file}</suiteXmlFile>
+					</suiteXmlFiles>
+					<argLine>
+						-javaagent:${com.browserstack:browserstack-java-sdk:jar}
+					</argLine>
+				</configuration>
 			</plugin>
 		</plugins>
 	</build>
 
 	<profiles>
 		<profile>
-			<id>single</id>
+			<id>sample</id>
```

Comment:
`sample-test`

Reply:
Not changing names as of now, keeping it same which was previously to minimize docs changes. Later, this can be picked as a task to standardize the names across all repos.